evm/utils/fixture_tests.py

```diff
@@ -205,8 +206,14 @@ def normalize_to_address(value):
 
 def robust_decode_hex(value):
     unprefixed_value = remove_0x_prefix(value)
-    if len(unprefixed_value) % 2:
-        return decode_hex(pad_left(unprefixed_value, len(unprefixed_value) + 1, b'0'))
+
+    if is_bytes(value):
```

Comment:
`remove_0x_prefix` will crash on a `bytes` value before we get here.

I think this method definition is equivalent to `robust_decode_hex = eth_utils.curried.hexstr_if_str(to_bytes)`, which is equivalent to: `to_bytes(hexstr=value) if is_string(value) else to_bytes(value)`